### PR TITLE
CLI formatting improvements

### PR DESCRIPTION
**Pull request by silhouettes**
- Add package source information (what deployment/version each release is promoted/rolled back from)
- Format dates to be human readable (this matches what GitHub does for commits - it shows 'a few seconds ago', '2 hours ago', etc up until one month ago, at which it starts displaying 'Nov 10', etc)
- Colorize fields for more readability

How it looks:
image

### CODE REVIEW

**Comment by dtivel** [comment]
Looks good.

File: cli/script/command-executor.ts [hunk 1]
```diff
@@ -601,11 +605,24 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, packageH
...
+
+                if (releaseSource) {
+                    // Need to word-wrap internally because wordwrap is not smart enough to ignore color characters
+                    releaseTime += "\n" + chalk.magenta(`(${releaseSource})`).toString()
+                }
+
                 dataSource.push([
-                    packageObject.label,
-                    new Date(packageObject.uploadTime).toString(),
+                    chalk.green(packageObject.label),
```
Comment (dtivel) [inline cli/script/command-executor.ts]: I don't think this should be colored.

Comment (silhouettes) [inline cli/script/command-executor.ts]: I think I agree actually. I've changed it back.

**Comment by lostintangent** [comment]
LGTM!

**Pull request merged**